types/src/system/auction/providers.rs

```diff
@@ -43,10 +43,10 @@ pub trait StorageProvider {
     fn write_bid(&mut self, account_hash: AccountHash, bid: Bid) -> Result<(), Error>;
 
     /// Reads collection of [`UnbondingPurse`]s at account hash derived from given public key
-    fn read_withdraw(&mut self, account_hash: &AccountHash) -> Result<Vec<UnbondingPurse>, Error>;
+    fn read_unbond(&mut self, account_hash: &AccountHash) -> Result<Vec<UnbondingPurse>, Error>;
```

Comment:
This renaming (and `get_withdraw() -> get_unbonds()`) could have been introduced as a separate PR.